### PR TITLE
Changed GOOGLE_BUCKET_NAME to GOOGLE_STORAGE_BUCKET

### PR DESCRIPTION
**Pull request by willgoldby**
I'm having users pick their own database names so thought removing the comment made sense.

### CODE REVIEW

File: appengine/php72/getting-started/app.yaml [hunk 1]
```diff
@@ -2,10 +2,9 @@ runtime: php72
 instance_class: F2
 
 env_variables:
-    GOOGLE_BUCKET_NAME: ""
+    GOOGLE_STORAGE_BUCKET: ""
     # populate these to use the "mysql" or "postres" backends
     CLOUDSQL_CONNECTION_NAME: ""
     CLOUDSQL_USER: ""
     CLOUDSQL_PASSWORD: ""
-    ## Uncomment to give your database a name other than "bookshelf"
```
Comment (bshaffer) [inline appengine/php72/getting-started/app.yaml]: Why did this comment get removed? And also the `CLOUDSQL_DATABASE_NAME` should remain commented out, as it defaults to `bookshelf` so it isn't required.

Comment (willgoldby) [reply appengine/php72/getting-started/app.yaml]: In the tutorial, users create a new database and name it, so I was thinking all users would need to uncomment it out... What are your thoughts on that?

Comment (bshaffer) [reply appengine/php72/getting-started/app.yaml]: Well, I wanted to keep it simple and have it default to "bookshelf". The tutorial could suggest this, or another name like `getting_started_php` if you prefer, but I think it's nice to keep it optional.

Comment (willgoldby) [reply appengine/php72/getting-started/app.yaml]: Sounds good to me. I'll just make a note in the tutorial. :)

**Pull request closed without merging**